### PR TITLE
Added coding guidelines.

### PR DESCRIPTION
**Pull request by chhavip**
Coding guidelines according to what I have learned so far, will add more as needed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,3 +27,11 @@ and self-esteem by navigating the life of their avatar!
 ## Documentation of PowerUp (Android)
 Here's the link to the official documentation:
 [Visit Documentation!](http://chetnagsocpowerupandroid.blogspot.in/2015/05/database-design.html)
+
+##Coding Guidelines
+1. Don't use magic numbers or hard-coded strings. Put them in dimens.xml or strings.xml
+2. Class names should be in CamelCase. Name activities with names including Activity so it's easier to know what they are.
```
Comment (alma-castant) [inline README.md]: You can use `1.` for all elements in the list and that will generate the right order so changes are easier

File: README.md [hunk 2]
```diff
@@ -27,3 +27,11 @@ and self-esteem by navigating the life of their avatar!
...
+3. Include spaces between parameters when you call a method for example: Intent(MainActivity.this, GameActivity.class).
+4. Give relevant names to buttons and other resources.
```
Comment (alma-castant) [inline README.md]: Quote code with between ` elements so it's easier to read

File: README.md [hunk 3]
```diff
@@ -27,3 +27,11 @@ and self-esteem by navigating the life of their avatar!
 ## Documentation of PowerUp (Android)
 Here's the link to the official documentation:
 [Visit Documentation!](http://chetnagsocpowerupandroid.blogspot.in/2015/05/database-design.html)
+
+##Coding Guidelines
+1. Don't use magic numbers or hard-coded strings. Put them in dimens.xml or strings.xml
+2. Class names should be in CamelCase. Name activities with names including Activity so it's easier to know what they are.
+3. Include spaces between parameters when you call a method for example: Intent(MainActivity.this, GameActivity.class).
+4. Give relevant names to buttons and other resources. 
+5. Use @id instead of @+id when referring to resources that have been already created in xml files.
```
Comment (alma-castant) [inline README.md]: Same: quote `@id`and `@+id`

**Comment by alma-castant** [comment]
I left some comments. I am also thinking about whether the README file is the best place for this or if another `.md` file or the wiki would be best

**Comment by chhavip** [comment]
I was thinking the same, wiki or readme, you let me know what is more
appropriate and I'll make the changes and post the guide there.

Chhavi P. Gupta

**Comment by alma-castant** [comment]
I've thought about it and this file is probably fine for now. We can move it to a different file if the list eventually gets bigger. Feel free to make the changes here

**Comment by chhavip** [comment]
I have created a new pull request with the proposed changes.

**Pull request closed without merging**